### ISSUE TITLE
0.8 to 0.9 upgrade results in ReHUD starting but no UI

### ISSUE THREAD

**Issue by duartevasco** [Yuvix25/ReHUD#8]
**Describe the bug**
I upgraded from 0.8 in place (just ran the setup)
When I start the 0.9, nothing happens visually, but when I go to Task Manager there are severa ReHUD processes running although nothing is visible

**To Reproduce**
1. install 0.8
2. have some user data, etc.
3. upgrade to 0.9 by running the setup 
=> 0.9 starts (seen in Task manager), but no UI is visible, nor do the widgets show up when playing R3E

**Expected behavior**
=> 0.9 should start and behave as 0.8

**Screenshots**

**ReHUD Version:** (fill here)
[ReHUD.log](https://github.com/user-attachments/files/16738941/ReHUD.log)

0.8 to 0.9 upgrade

**Additional context**

**Log file**
Attach the `ReHUD.log` file which can be found by clicking on "Show Log File" in the ReHUD settings window.

**Comment by Yuvix25** [comment]
@duartevasco Hi, another user experienced the same issue, can you zip your Documents/ReHUD folder and send it here?
After you do that, rename that folder to anything else (ReHUD-1) for example, and the app should load correctly.

**Comment by Yuvix25** [comment]
Ok, after investigating the other user's files, I found the issue. There was some bug with casting operation gone wrong.
As a temporary workaround, modify your settings.json file in your Documents/ReHUD folder and add a `"framerate": 60` entry. For example if the settings file contained:
```json
{"check-for-updates": false, "radarRange":12}
```
Change it to:
```json
{"check-for-updates": false, "radarRange":12, "framerate":60}
```
This should fix it for now, and I'll patch that bug in the next release.  
Thanks for bringing this to my attention, and feel free to re-open this issue if the fix doesn't work.

**Comment by Yuvix25** [comment]
Fixed with d4ac6b850f36997084aa57c7688bbed01b5ebd83.
